src/mui/field/AmountField.js

```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes } from 'react';
+import get from 'lodash.get';
+
+/**
+ * @example
+ * <AmountField source="total" currency="€" decimals={2} />
+ */
+const AmountField = ({ record, source, currency, decimals, elStyle }) => record ?
+    <span style={elStyle}>{currency}{get(record, source).toFixed(decimals)}</span> :
```

Comment:
Some currencies require their symbol to be put **after** the number (`€` for example)